packages/ckb/src/utils/ckb-tx.spec.ts

```diff
@@ -31,6 +32,9 @@ describe('ckb tx utils', () => {
     };
     const capacity = calculateRgbppCellCapacity(xudtType);
     expect(BigInt(254_0000_0000)).toBe(capacity);
+
+    const actual = calculateRgbppCellCapacity();
+    expect(actual).toBe(BigInt(254_0000_0000));
```

Comment:
What is the purpose of this test?

Reply:
The `calculateRgbppCellCapacity` function allows calling without parameters

Reply:
understood.